src/views/PeopleSearch/index.js

```diff
@@ -589,15 +408,49 @@ class PeopleSearch extends Component {
               }
             </Media>
           ),
-          header: this.makeHeader(),
+          header: peopleSearchHeader,
         });
       }
     }
   }
 
-  getDate = () => {
-    return new Date();
-  };
+  async updateURL() {
```

Comment:
The verbosity of this method suggests that we could probably improve the code a lot by storing search values in an object of the form `{ firstName: value, lastName: otherValue, ...}`. Then, rather than repeating the same process for every searchValue, we could simply map over the values: 
```
const searchParameters = Object.entries(this.state.searchValues)
  .map(([key, value]) => value ? `${key}=value` : '') // [ 'firstName=value', 'state=texas']
  .join('&') // 'firstName=value&state=texas' - note the lack of trailing &

if (searchParameters) {
  this.props.history.push(`?${searchParameters}`)
}
```

Reply:
You can just do `this.setState({searchValues: {...searchValues, firstName: e.target.value}})` right?

Reply:
Ah, right, I get what you mean. Because `this.setState` only does shallow object comparison for `state`, having a nested object that gets overwritten will cause a re-render every time. I think that shouldn't affect performance negatively though, since there's so few of them. That's something that can be solved when this page is refactored to use hooks.